EFSAdvent/data/actorinfo/BRRY.txt

```diff
@@ -1,4 +1,4 @@
-BRRY (Buried rupee)
+BRRY (Buried rupee) No GBA sprite
```

Comment:
Are you sure?
I thought it was displayed even if something is buried on the GBA.